runsc/main.go

```diff
@@ -339,10 +341,18 @@ func main() {
 			// signal to self?
 			os.Exit(128 + int(ws.Signal()))
 		}
-		os.Exit(ws.ExitStatus())
+		if *coverageFilesDirectory != "" {
+			return
+		} else {
+			// If we exit here, no coverage files will be generated.
```

Comment:
Please add this comment to the coverage case rather than the standard case. Does this affect test failure/success if we do not exit with the proper status?